composer.json

```diff
@@ -1,5 +1,7 @@
 {
     "require": {
-        "phpunit/phpunit": "^5"
+        "phpunit/phpunit": "^5",
+        "twig/twig": "^1.0",
```

Comment:
OK if the current version of Twig is out of date the one in the lib directory should be be overwritten with the new version, and the files for the new version committed in this repo. Obviously this isn't how you should do it, but removing dependencies from lib would mean having a working build process to deploy it and at this time I'm not ready to do that.

See the peculiar thing about lainchan is that it assumes the data directories it writes to are relative to it's source root. This makes sense in the context of a cgi environment but on NixOS a built package resides on a read-only file system. So I have to figure out if I want to have the nix derivation create symlinks or just add support for lainchan to keep it's code and data separate.

So for now we need to be able to deploy this repo without any build system. (not that you can't still add build support, but the products have to be committed).

Reply:
Right now this implementation requires both versions of Twig to exist in parallel. This is because the Tinyboard Twig extension uses the old version. I will look into rewriting it so we can include the new version of Twig in lib.

Reply:
Wait, what uses the new version?

Reply:
Everything but the Tinyboard extension.